### PR TITLE
Android deletion updates

### PR DESCRIPTION
**Pull request by burtonemily**
Updates tests that deal with any kind of message deletion, as per androids updates

### CODE REVIEW

File: run/types/testing.ts [hunk 1]
```diff
@@ -359,13 +361,10 @@ export type Id =
...
+  | 'network.loki.messenger:id/search_result_title'
+  | 'Error message'
   | 'network.loki.messenger:id/action_apply';
 
-export type testRisk = 
-  | 'high' 
-  | 'medium' 
-  | 'low' 
-  | 'undefined';
+export type testRisk = 'high' | 'medium' | 'low' | 'undefined';
```
Comment (Bilb) [inline run/types/testing.ts:370]: do we really need `'undefined'` and not `undefined` ? this seems like an issue.

File: package.json [hunk 2]
```diff
@@ -46,7 +45,7 @@
     "@playwright/test": "^1.45.1",
     "appium": "^2.4.1",
     "appium-uiautomator2-driver": "patch:appium-uiautomator2-driver@npm%3A3.7.3#~/patches/appium-uiautomator2-driver-npm-3.7.3-24e944f6c5.patch",
-    "appium-xcuitest-driver": "^7.26.0",
```
Comment (Bilb) [inline package.json:49]: not sure why a downgrade is needed?

Comment (burtonemily) [reply package.json:49]: will revert this, forgot to do this after debugging the error earlier

File: run/types/testing.ts (hunk 1 above)
Comment (burtonemily) [reply run/types/testing.ts:370]: This was from Miki's PR, can change it to not have quotes

Comment (Bilb) [reply run/types/testing.ts:370]: yeah we need it without quotes I reckon.

Comment (burtonemily) [reply run/types/testing.ts]: done

File: .eslintrc.js [hunk 3]
```diff
@@ -1,20 +0,0 @@
...
-    tsconfigRootDir: __dirname,
-    project: ["./tsconfig.json"],
-  },
-  rules: {
-    "@typescript-eslint/no-floating-promises": "error",
-    "@typescript-eslint/no-unsafe-member-access": "off",
-    "@typescript-eslint/no-explicit-any": "off",
-    "@typescript-eslint/no-unsafe-call": "off",
-    "@typescript-eslint/no-unsafe-assignment": "off",
-    "@typescript-eslint/no-inferrable-types": "off",
```
Comment (Bilb) [inline .eslintrc.js:18]: those rules are missing in the updated .mjs file below.

Comment (Bilb) [reply .eslintrc.js:18]: same for `plugin:@typescript-eslint/recommended-requiring-type-checking`

File: run/localizer/component/Localizer.ts [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
-    `<(?:${supportedFormattingTags.join('|')})>.*?</(?:${supportedFormattingTags.join('|')})>|<(?:${supportedSelfClosingFormattingTags.join('|')})\\/>`,
-    'g'
-  );
-}
+// function createSupportedFormattingTagsRegex() {
+//   return new RegExp(
+//     `<(?:${supportedFormattingTags.join('|')})>.*?</(?:${supportedFormattingTags.join('|')})>|<(?:${supportedSelfClosingFormattingTags.join('|')})\\/>`,
+//     'g'
+//   );
+// }
```
Comment (Bilb) [inline run/localizer/component/Localizer.ts:11]: can be removed completely

**Review by Bilb — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 3 above)
Comment (burtonemily) [reply .eslintrc.js:18]: added into mjs file

File: run/test/specs/utils/set_disappearing_messages.ts [hunk 5]
```diff
@@ -34,7 +33,11 @@ export const setDisappearingMessage = async (
       await device.disappearRadioButtonSelected(platform, DISAPPEARING_TIMES.ONE_DAY);
     }
   } else if (enforcedType === 'Group' && timerType === 'Disappear after send option') {
-    await device.disappearRadioButtonSelected(platform, DISAPPEARING_TIMES.OFF);
+    if (platform === 'ios') {
+      await device.disappearRadioButtonSelected(platform, DISAPPEARING_TIMES.OFF_IOS);
+    } else {
+      await device.disappearRadioButtonSelected(platform, DISAPPEARING_TIMES.OFF_ANDROID);
+    }
```
Comment (Bilb) [inline run/test/specs/utils/set_disappearing_messages.ts:40]: this could be runOnAndroid/iOS instead of a if/else

**Review by Bilb — APPROVED** [review]

**Pull request merged**